cli/cmd/sync.go

```diff
@@ -233,6 +234,11 @@ func sync(cmd *cobra.Command, args []string) error {
 		if err != nil {
 			return enrichClientError(managedplugin.Clients{}, []bool{source.RegistryInferred()}, err)
 		}
+		latestVersion, err := sourcePluginClient.FindLatestPluginVersion(ctx, managedplugin.PluginSource)
```

Comment:
I think `managedplugin.NewClient(ctx, managedplugin.PluginSource, cfg, opts...)` downloads the plugins binary first which is not needed and will fail it auth is not provided I believe